wp-autoupdates.php

```diff
@@ -21,7 +21,7 @@
 	die( 'Invalid request.' );
 }
 
-define( 'WP_AUTO_UPDATES_VERSION', '0.8.0' );
+define( 'WP_AUTO_UPDATES_VERSION', '0.8.1' );
```

Comment:
forgot `package.json` and `package-lock.json` again :-)

Reply:
Damn… thanks Paul :D